helm/frost-server/values.yaml

```diff
@@ -245,23 +246,25 @@ frost:
         message_size: 8092
 
     ingress:
-      enabled: true      
+      enabled: true
+      ## Set to true to use the same ingress as the HTTP service, otherwise set to false
+      ## no dedicated ingress will be created for the MQTT service
+      useSameAsHttp: false
       ## Set a specific ingress-class for the ingress ([default] not set)
-      # className: azure-application-gateway
-      ## Defines the type of ingress-Controller in use (nginx [default], agic or traefik) 
-      # ingressProvider: agic
+      # ingressClassName: nginx
+      ## Defines the type of ingress-Controller in use (kubernetes-nginx [default], nginx-nginx, agic, none) 
+      ## Use "none" to disable render predefined annotations for this service
+      # ingressProvider: "nginx-nginx"
       ## Additional annotations
-      annotations:
-        nginx.ingress.kubernetes.io/proxy-read-timeout: "3600"
-        nginx.ingress.kubernetes.io/proxy-send-timeout: "3600"
-        # cert-manager.io/cluster-issuer: letsencrypt
-        # cert-manager.io/issue-temporary-certificate: "true"
+      # annotations:
+      #   cert-manager.io/cluster-issuer: letsencrypt
+      #   cert-manager.io/issue-temporary-certificate: "true"
       tls:
         enabled: false
         secretName: []
 
     # FROST-Server MQTT business settings
-    serviceHost: "" # default URI is the same then frost.http.serviceHost"
+    serviceHost: "mqtt.mydomain.com" # default URI is the same then frost.http.serviceHost"
```

Comment:
If there is a value here by default, the default is no longer the same as `frost.http.serviceHost`.